src/main/java/org/jboss/xavier/analytics/pojo/output/AnalysisModel.java

```diff
@@ -32,11 +32,11 @@
     )
     private Long id;
 
-    @OneToOne(mappedBy = "analysis", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OneToOne(mappedBy = "analysis", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
     @JsonIgnore
     private InitialSavingsEstimationReportModel initialSavingsEstimationReportModel;
 
-    @OneToMany(mappedBy = "analysis", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OneToMany(mappedBy = "analysis", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
1. After we change from EAGER to LAZY the deprecated method `addWorkloadInventoryReportModel` is not going to work anymore, which is expected and ok. I suggest to delete this deprecated method in this PR. https://github.com/project-xavier/xavier-integration/blob/292c2494f912d7289119b4a8af365265b117d057/src/main/java/org/jboss/xavier/integrations/jpa/service/AnalysisService.java#L69

Reply:
+1, if fact I dont understand why we did set it to deprecated and no other method in the code was using it.....